compose.yml

```diff
@@ -1,81 +1,155 @@
+      DB_IP: mysql
+    volumes:
+      - .:/app
+      - dotnet-tools:/root/.dotnet/tools
+
+  file-ingest:
+    container_name: maple2-file-ingest
+    image: mcr.microsoft.com/dotnet/sdk:8.0
+    working_dir: /app/Maple2.File.Ingest
+    command: dotnet run
+    depends_on:
+      - mysql
+    links:
+      - mysql
+    env_file:
+      - .env
+    environment:
+      DB_IP: mysql
+      MS2_DATA_FOLDER: /app/MapleStory2/Data
+    volumes:
+      - .:/app
+      - /home/user/MapleStory2/Data:/app/MapleStory2/Data
+
+  login:
     build:
       context: .
-      dockerfile: ./Maple2.Server.World/Dockerfile
-    image: maple2-world:dev
+      dockerfile: ./Maple2.Server.Login/Dockerfile
+    container_name: maple2-login
+    image: maple2/login
+    command: dotnet Maple2.Server.Login.dll
+    depends_on:
+      mysql:
+        condition: service_started
+      world:
+        condition: service_started
     ports:
-      - "21001:21001"
+      - 20001:20001
+    links:
+      - world
+    env_file:
+      - .env
     environment:
+      DB_IP: mysql
+      WORLD_SERVICE: world
       GAME_DB_CONNECTION: "Server=host.docker.internal;Database=game-server;User=root;Password=maplestory;oldguids=true"
       DATA_DB_CONNECTION: "Server=host.docker.internal;Database=maple-data;User=root;Password=maplestory;oldguids=true"
-      CHANNEL_SERVICE: "game-main,game-alt"
-      GAME_IP: "127.0.0.1"
-      LOGIN_IP: "127.0.0.1"
 
-  login:
+  world:
     build:
       context: .
-      dockerfile: ./Maple2.Server.Login/Dockerfile
-    image: maple2-login:dev
+      dockerfile: ./Maple2.Server.World/Dockerfile
+    container_name: maple2-world
+    image: maple2/world
+    command: dotnet Maple2.Server.World.dll
     depends_on:
-      - world
+      - mysql
     ports:
-      - "20001:20001"
-    links:
-      - "world"
+      - 21001:21001
+    env_file:
+      - .env
     environment:
+      DB_IP: "mysql"
       GAME_DB_CONNECTION: "Server=host.docker.internal;Database=game-server;User=root;Password=maplestory;oldguids=true"
       DATA_DB_CONNECTION: "Server=host.docker.internal;Database=maple-data;User=root;Password=maplestory;oldguids=true"
-      WORLD_SERVICE: "world"
-      LOGIN_IP: "127.0.0.1"
-      WEB_URI: "http://127.0.0.1"
+      CHANNEL_SERVICE: "game-main,game-alt"
 
   game-main:
     build:
       context: .
       dockerfile: ./Maple2.Server.Game/Dockerfile
-    image: maple2-game:dev
+    image: maple2/game
     depends_on:
+      - mysql
       - world
     ports:
-      - "20002:20002"
-      - "21002:21002"
+      - 20002:20002
+      - 21002:21002
+    env_file:
+      - .env
     environment:
+      DB_IP: "mysql"
       GAME_DB_CONNECTION: "Server=host.docker.internal;Database=game-server;User=root;Password=maplestory;oldguids=true"
       DATA_DB_CONNECTION: "Server=host.docker.internal;Database=maple-data;User=root;Password=maplestory;oldguids=true"
       GAME_CHANNEL: 1
       GAME_PORT: 20002
       GRPC_CHANNEL_PORT: 21002
```

Comment:
these can be removed on both game containers

Reply:
@AngeloTadeucci are you sure?

They are used here:
https://github.com/AngeloTadeucci/Maple2/blob/14703254b9a4a41e73d21156ec32214c1f164983/Maple2.Server.Core/Constants/Target.cs#L30-L45

Reply:
it was removed in this commit: https://github.com/AngeloTadeucci/Maple2/commit/41f7a7f78fcbf5c6846fd1bd5e637a23af6cc16d

i updated the base branch